src/en/news/blog/2025/v18-2-6-reef-released/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "v18.2.6 Reef released"
+date: "2025-04-21"
+author: "Yuri Weinstein"
+tags:
+  - "release"
+  - "reef"
+---
+
+This is the sixth backport (hotfix) release in the Reef series. We recommend that all users update to this release.
+
+## Notable Changes
+
+- ceph-volume: A bug related to cryptsetup version handling has been fixed.
+
+  Related tracker: https://tracker.ceph.com/issues/66393
+
+- RADOS: A bug related to IPv6 support is now fixed.
+
+  Related tracker: https://tracker.ceph.com/issues/67517
+
+## Changelog
+
+- ceph-volume: fix regex usage in `set\_dmcrypt\_no\_workqueue` ([pr#62791](https://github.com/ceph/ceph/pull/62791), Matt1360)
+
+- common/pick\_address: Add IPv6 support to is\_addr\_in\_subnet ([pr#62814](https://github.com/ceph/ceph/pull/62814), rzarzynski)
```

Comment:
Why the backslashes?  I think they will get rendered, suggest taking them out

Reply:
I have no problems removing them, but they came from the `ceph-release-notes` script
If the script does not do it right, we need to fix it.  Otherwise, for a large RN it will be problematic manually fixing 

@batrick @ljflores can you please comment?

Command line:

`LANG=en_US.UTF-8 python3 ~/wip_main/src/script/ceph-release-notes -r tags/v18.2.5..origin/reef-release $(git rev-parse --show-toplevel) --token $(cat ~/.github_token) --cherry_picks '62791,62814' --markdown`